src/components/Auth.jsx

```diff
@@ -36,71 +41,43 @@ user is redirected to the URL they attempted to login from initially.
 export class Auth extends React.Component {
```

Comment:
We still use it for snapshot testing, which won't work with connected components